python/graphstorm/gconstruct/file_io.py

```diff
@@ -80,6 +80,9 @@ def read_data_csv(data_file, data_fields=None, delimiter=','):
     dict of Numpy arrays.
     """
     data = pd.read_csv(data_file, delimiter=delimiter)
+    assert data.shape[0] > 0, \
+        f"{data_file} has an empty data. The data frame shape is {data.shape}"
```

Comment:
if the file is empty, what is the shape of the data?

Reply:
It still contains the column names.